proj/src/main/webapp/static/signup.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Welcome</title>
+    <!-- Bootstrap: Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <!-- Bootstrap: jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Bootstrap: Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="../style.css">
+    <!-- Font  -->
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container screen">
+      <div class="row fullscreen">
+        <!-- title  desktop:left    mobile:none -->
+        <div class="col-sm-8 info-graphic">
+          <div class="header">
+            <h1> camaraderie </h1>
```

Comment:
Is there a reason for the spacing around tags here and below?  It's not a big deal, but I don't think it really does anything.

Reply:
Sorry am I using the wrong styling guide?  The spacing I was using was mainly for organization. Please let me know how ti make it cleaner.

Reply:
Sorry, I meant the space between the tags and the content, like around the word "camaraderie".  That's nonstandard.

Reply:
Haha gotcha. I didn't even notice it, it wasn't intentional so I'll change the spaces